### ISSUE TITLE
Google Smart Home now supports sensors

### ISSUE THREAD

**Issue by JSa1987** [oznu/homebridge-gsh#77]
**Is your feature request related to a problem? Please describe:**
Home bridge supports Temperature Sensors but these are currently not supported by homebridge-gsh

**Describe the solution you'd like:**
Support for sensors has been added to Google Smart Home (https://developers.google.com/assistant/smarthome/traits).
As per https://developers.google.com/assistant/smarthome/traits/temperaturecontrol seems that a action.devices.traits.TemperatureControl device could be used to monitor a temperature sensor setting the queryOnlyTemperatureControl attribute to True.
Would it be possible to use this to make the Homebridge temperature sensors available available in google home? 

**Describe alternatives you've considered:**
The current alternative is to use Apple HomeKit, but the limits the accessibility to temperature sensors to Apple devices.

**Additional context:**
Would be nice to have the temperature sensors accessible also from Google assistant via Homebridge.

**Comment by oznu** [comment]
Added support for:

* Temperature Sensor
* Humidity Sensor

Motion and Occupancy Sensors still look like they won't implement nicely. Some of the other sensor types should be possible to map, such as "Air Quality" or "Smoke".

**Comment by JSa1987** [comment]
Hello,

I updated the plug in and was able to add temperature sensors to my google home app. However, it seems that when the google assistant tries to retrieve the temperature data this doesn't work.

Below is what shows up in the logs when I ask the google assistant for the temperature of a room:

```
[10/5/2020, 1:26:23 PM] [Google Smart Home] Received QUERY intent
[10/5/2020, 1:26:23 PM] [Google Smart Home] {
  "inputs": [
    {
      "intent": "action.devices.QUERY",
      "payload": {
        "devices": [
          {
            "customData": {
              "aid": 3,
              "iid": 8,
              "instanceIpAddress": "192.168.1.2",
              "instancePort": 51558,
              "instanceUsername": "0E:B2:D4:EE:E3:5C"
            },
            "id": "c613fd26da26ebea210de54274d112530e41bcf6e1477e83120784f894235c4f"
          }
        ]
      },
      "requestId": "8419342326635291713"
    }
  ],
  "requestId": "8419342326635291713"
}
[10/5/2020, 1:26:23 PM] [HttpWebHooks] Getting current state for 'sensor39'...
[10/5/2020, 1:26:23 PM] [HttpWebHooks] State for 'sensor39' is '18.60'
[10/5/2020, 1:26:23 PM] [Google Smart Home] { c613fd26da26ebea210de54274d112530e41bcf6e1477e83120784f894235c4f: { online: true, temperatureAmbientCelsius: 18.6 } }
```

Seems that Homebridge receives that request from google assistant, retrieves the temperature from the correct sensor and tries to pass this back to google assistant... but seems this last step is not successful.

**Comment by oznu** [comment]
Temp sensor should be fixed in 2.0.0

**Comment by JSa1987** [comment]
I have just updated home bridge and it's now working great!
